core/src/main/java/org/jdbi/v3/core/statement/SqlStatement.java

```diff
@@ -179,10 +181,36 @@ public This bindBean(Object bean) {
      *
      * @return modified statement
      */
+    @SuppressWarnings("deprecation")
     public This bindBean(String prefix, Object bean) {
         return bindNamedArgumentFinder(new BeanPropertyArguments(prefix, bean));
     }
 
+    /**
+     * Binds named parameters from object properties on the argument.
+     *
+     * @param pojo source of named parameter values to use as arguments
+     *
+     * @return modified statement
+     */
```

Comment:
Should document that the pojo type must already be registered, and add `@see` to the possible registration methods. Ditto for the next method.